libs/nextjs/src/helpers.ts

```diff
@@ -139,13 +123,82 @@ export async function refreshToken(
   }
 }
 
-export function addToCookies(cookieValue: string, res: ServerResponse) {
+type CreateCookieArguments = {
+  cookieName?: string,
+  session: string,
+  expires: CookieSerializeOptions['expires'],
+  isSecured: CookieSerializeOptions['secure'],
+  cookieDomain?: CookieSerializeOptions['domain'],
+  httpOnly?: CookieSerializeOptions['httpOnly'],
+  path?: CookieSerializeOptions['path']
+}
+const COOKIE_MAX_LENGTH = 4096
+export function createCookie(
+  {
+    cookieName = fronteggConfig.cookieName,
+    session,
+    expires,
+    isSecured,
+    cookieDomain = fronteggConfig.cookieDomain,
+    httpOnly = true,
+    path = '/'
+  }: CreateCookieArguments) {
+  const options = {
```

Comment:
can we get the options with `...rest` for the one the we don't manipulate for `options` ?

Reply:
probably you did it for the defaults, right? we can try to manage it in options as well, not sure what better

Reply:
I will fix it the next pr